src/content/decentralized-social-networks.md

```diff
@@ -0,0 +1,92 @@
+
+Many blockchain-based social platforms have native tokens that power monetization in absence of advertising revenue. Users can buy these tokens to access certain features, complete in-app purchases, or tip their favorite content creators. 
+
+## Decentralized social networks on Ethereum {#Ethereum-social-media}
+
+The Ethereum network has become the preferred tool for developers creating decentralized social media networks owing to the popularity of its tokens (ERC-20/ERC-721) and its massive user base. Here are some examples of Ethereum-based social networks:
+
+- Peepeth
+
+[Peepeth](https://peepeth.com/) is a blockchain-based microblogging site similar to Twitter. The dApp runs on the Ethereum blockchain and uses the InterPlanetary File System (IPFS) to store user data.
+
+Users can send short messages called "Peeps", which cannot be deleted or modified. You can collect tips or tip anyone on the platform in ether (ETH) without leaving the app.
+
+- Mirror
+
+[Mirror](https://mirror.xyz/) is a decentralized and user-owned content distribution platform similar to Medium. Users can sign up by connecting their wallets or providing an Ethereum Name Service (ENS) username.
+
+Posts published on Mirror are permanently stored on Arweave, a decentralized storage platform. Users can earn money by selling their articles as [non-fungible tokens (NFTs)](https://ethereum.org/en/nft/).
+
+- MINDS
+
+[MINDS](https://www.minds.com/) is considered one of the more popular blockchain social media platforms out there. It works like Facebook, and has racked up millions of users already.
+
+Users use the platform's native ERC-20 token $MIND to pay for items. You can also earn $MIND tokens by publishing popular content, contributing to the ecosystem, and accepting advertising offers.
+
+## Ethereum and legacy social media platforms {#Ethereum-and-legacy-social-media-networks}
+
+Decentralized social media platforms aren't the only ones trying to incorporate blockchain technology. Traditional sites are warming up to integrating Ethereum into their infrastructure.
+
+For instance, Reddit has [touted Community Points](https://cointelegraph.com/news/reddit-to-reportedly-tokenize-karma-points-and-onboard-500m-new-users), which are ERC-20 token that users can earn by posting quality content and contributing to the online community. You can redeem these tokens to gain exclusive privileges and perks.
+
+Early this year, Twitter Blue [rolled out support for NFTs](https://mashable.com/article/twitter-blue-nft-profile-picture), allowing users to connect their wallets and display NFTs as profile pictures. The social media company has also [announced plans](https://www.theverge.com/2021/8/16/22627435/twitter-bluesky-lead-jay-graber-decentralized-social-web) to create a decentralized social network in future.
+
+## Use Ethereum-based social networks {#Use-Ethereum-social-media}
+
+- [Indorse](https://indorse.io/)
```

Comment:
What is this? A protocol? Their website isn't very clear. Either way, we should focus on dapps as that as what we're describing above—clicking on this will only confuse readers.

```suggestion
```

Reply:
I agree on this point. It was initially marketed as some sort of "blockchain-based LinkedIn" where jobseekers can meet companies, but I think they've shifted focus. Now, the business model is offering decentralized code reviews.